pkg-new/hostutils/initialize.go

```diff
@@ -16,16 +17,34 @@ type InitForInstallOptions struct {
 	ServiceCIDR  string
 }
 
-func (h *HostUtils) ConfigureForInstall(ctx context.Context, rc runtimeconfig.RuntimeConfig, opts InitForInstallOptions) error {
+func (h *HostUtils) ConfigureHost(ctx context.Context, rc runtimeconfig.RuntimeConfig, opts InitForInstallOptions) error {
+	// update process env vars from the runtime config
+	os.Setenv("KUBECONFIG", rc.PathToKubeConfig())
+	os.Setenv("TMPDIR", rc.EmbeddedClusterTmpSubDir())
```

Comment:
Why does this belong in this function?

Reply:
it doesn't, just moved it to hostutils

Reply:
setting the env is unrelated to configuring the host. i am referring only to that. duplicating is fine.